script/deploy/mainnet/011_OgnOgvMigrationScript.sol

```diff
@@ -54,102 +64,94 @@ contract OgnOgvMigrationScript is BaseMainnetScript {
+        govProposal.action(
             Addresses.TIMELOCK,
             "revokeRole(bytes32,address)",
             abi.encode(timelock.EXECUTOR_ROLE(), Addresses.GOVERNOR_FIVE)
         );
 
-        govFive.action(Addresses.OUSD_BUYBACK, "upgradeTo(address)", abi.encode(Addresses.OUSD_BUYBACK_IMPL)); // Todo, use latest deployed address
-        govFive.action(Addresses.OUSD_BUYBACK, "setRewardsSource(address)", abi.encode(ognRewardsSourceProxy));
-        govFive.action(Addresses.OETH_BUYBACK, "upgradeTo(address)", abi.encode(Addresses.OETH_BUYBACK_IMPL)); // Todo, use latest deployed address
-        govFive.action(Addresses.OETH_BUYBACK, "setRewardsSource(address)", abi.encode(ognRewardsSourceProxy));
-
-        // Mint token proposal from OGN governance
-        IMintableERC20 ogv = IMintableERC20(Addresses.OGV);
-        // Mint additional OGN, will get returned after starting migration
-        uint256 ognToMint = ((ogv.totalSupply() * 0.09137 ether) / 1 ether) + 1_000_000 ether;
-
-        address[] memory targets = new address[](1);
-        string[] memory sigs = new string[](1);
-        bytes[] memory calldatas = new bytes[](1);
-
-        // OGN Gov 1: Mint OGN
-        targets[0] = Addresses.OGN;
-        sigs[0] = "mint(address,uint256)";
-        calldatas[0] = abi.encode(deployedContracts["MIGRATOR"], ognToMint);
-
-        govFive.action(
-            Addresses.OGN_GOVERNOR,
-            "propose(address[],string[],bytes[],string)",
-            abi.encode(targets, sigs, calldatas, "")
+        // Upgrade buyback contracts & configure them
+        govProposal.action(Addresses.OUSD_BUYBACK, "upgradeTo(address)", abi.encode(Addresses.OUSD_BUYBACK_IMPL));
+        govProposal.action(Addresses.OUSD_BUYBACK, "setRewardsSource(address)", abi.encode(ognRewardsSourceProxy));
+        govProposal.action(Addresses.OETH_BUYBACK, "upgradeTo(address)", abi.encode(Addresses.OETH_BUYBACK_IMPL));
+        govProposal.action(Addresses.OETH_BUYBACK, "setRewardsSource(address)", abi.encode(ognRewardsSourceProxy));
+
+        // Mint OGN required
+        govProposal.action(
+            Addresses.OGN, "mint(address,uint256)", abi.encode(deployedContracts["MIGRATOR"], OGN_TO_MINT)
         );
 
-        if (!isForked) {
-            govFive.printTxData();
-        }
-    }
+        // Transfer in excess OGN from Multisig (for migration)
+        govProposal.action(
+            Addresses.OGN,
+            "transferFrom(address,address,uint256)",
+            abi.encode(Addresses.GOV_MULTISIG, deployedContracts["MIGRATOR"], EXTRA_OGN_FOR_MIGRATION)
+        );
 
-    function _fork() internal override {
-        // Simulate execute on fork
-        govFive.execute();
+        // Transfer in OGN from Multisig (for rewards)
+        govProposal.action(
+            Addresses.OGN,
+            "transferFrom(address,address,uint256)",
+            abi.encode(Addresses.GOV_MULTISIG, deployedContracts["OGN_REWARDS_SOURCE"], EXTRA_OGN_FOR_REWARDS)
```

Comment:
I think we are starting with just seven days worth of rewards on the rewards contract.